tests/affiliatedcertification/affiliated_certification_suite_test.go

```diff
@@ -50,4 +58,10 @@ var _ = AfterSuite(func() {
 	By("Remove reports from report directory")
 	err = globalhelper.RemoveContentsFromReportDir()
 	Expect(err).ToNot(HaveOccurred())
+
+	By("Re-label operator used in other suites")
```

Comment:
the reason there's no Expect(err).ToNot(HaveOccurred()) on this or the un-labeling above is that the cloudcasa operator is not installed on every ocp version, so we don't want the suite to fail if it's not there to be labeled.

Reply:
I don't like too much this approach. I think you should only re-apply the label if the operator actually had it before your test suite.

Reply:
Added a check :+1: